lib/livebook_web/live/session_live/secrets_component.ex

```diff
@@ -224,14 +224,20 @@ defmodule LivebookWeb.SessionLive.SecretsComponent do
     else
       {:error, changeset} ->
         {:noreply, assign(socket, changeset: changeset)}
+
+      {:transport_error, error} ->
```

Comment:
Maybe we could render the page again as is but with a flash message? I am not sure if we should redirect. Keeping on the page is easier to retry.

Reply:
It doesn't lose the input data, so a `patch` is fine.